src/scss/pages/_all.scss

```diff
@@ -13,4 +13,6 @@
 @import './footer';
 @import './header';
 @import './navigationMenu';
-@import './settings';
+@import './signup';
+@import './privacyPolicy';
+
```

Comment:
So it looks like the `settings` scss file reference got removed here during the merge conflict resolution, so let's add that back

Reply:
Added back and good to go!